test/fixtures/dialects/tsql/declare_table_type.sql

```diff
@@ -0,0 +1,7 @@
+DECLARE @MyTableVar TABLE(  
+    EmpID INT NOT NULL,  
+    OldVacationHours INT,  
+    NewVacationHours INT,  
+    ModifiedDate DATETIME,
+    PRIMARY KEY (EmpID)
+);
```

Comment:
Could you add a final newline here?

```suggestion
);

```